.prettierrc

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Can you move this config into `.eslintrc.js`?  (can have it in both places, but having it only here means eslint doens't do quite the right thing)

Reply:
Sure. I had it there before, but opted for an rc. I can swap it back.